scripts/test/shared.py

```diff
@@ -92,6 +92,11 @@ def parse_args(args):
         '--filter', dest='test_name_filter', default='',
         help=('Specifies a filter. Only tests whose paths contains this '
               'substring will be run'))
+    parser.add_argument(
+        '--auto-initial-contents', dest='auto_initial_contents',
+        action='store_true', default=False,
+        help='Select important initial contents automaticaly in fuzzer. '
+             'Default: disabled.')
 
```

Comment:
This is what we discussed yesterday, I guess - yeah, it's unfortunate that the current code doesn't let a specific script add an option. I guess we'd need to make a major refactoring to manually parse the args instead of automatically, so there is a chance to add more arguments. I guess for now this is ok, but maybe someone else will have an idea. (If not, we should probably add a comment here that this is just for `fuzz_opt.py`)

Reply:
Yeah this is unfortunate; I tried my hand at refactoring this for.. like 10 mins and stopped. For the moment I added a comment.

Reply:
It would be nice if `shared.py` just created the `parser` object and added the standard options to it, but then returned it back up to the main script instead of going ahead and parsing the arguments. That would be a large refactoring, though. Maybe a simpler change for now would be to add an optional parameter to `parse_args` that takes a lambda to be applied to the `parser` object (for example to add more options) before parsing.

Reply:
That's an option, but note that this is a preexisting problem; fuzz_opt.py only used two of the default options and others were ignored. Other scripts also use a subset of options each. So `parse_args` was already a big superset of all options taken by all scripts, so I thought it'd be weird if we make a parameter only for this option.

Also, even though this option is shown when you run `--help`, scripts under scripts/ directory are mostly used by developers, so the normal users are not gonna see it when they run things like `wasm-opt`. Actually there is an option to hide an option so that it doesn't show up in `--help`, but then it will need a separate comment on what it does, so thought it wasn't worth it.

And yeah refactoring seems to be a daunting task..